src/app/screens/Options/TestConnection/index.tsx

```diff
@@ -113,9 +117,35 @@ export default function TestConnection() {
                 </div>
 
                 <p className="mt-6 dark:text-gray-400"></p>
-                <p className="mt-6 dark:text-neutral-400">{t("ready")}</p>
```

Comment:
but now nothing is shown if a node is not required. 

we should keep the success then.

Reply:
hmm. sry! that was an important edge case. fixed